ovn-tester/ovn_workload.py

```diff
@@ -185,15 +200,37 @@ def start(self, cluster_cfg):
             cluster_cfg.db_inactivity_probe // 1000,
         )
 
+    def calculate_node_remotes(
```

Comment:
This produces incorrect results if n_relays > 0.  Nodes in AZ-X should connect to all relays in AZ-X.  Now they just connect to the central IPs of AZ-X.

Reply:
@dceara ack, I will fix it